02-wrangling.Rmd

```diff
@@ -2,14 +2,14 @@
 
 ## Overview 
 
-This chapter will be centered around tools for cleaning and wrangling data that move data from its raw format into a format that is suitable for data analysis. They 
-will be presented in the context of a real world data science application, providing more practice working through a whole case study.
+This chapter will be centred around tools for cleaning and wrangling data that move data from its raw format into a suitable format for data analysis. They will be presented in the context of a real-world data science application, providing more practice working through a whole case study.
 
 ## Chapter learning objectives
 By the end of the chapter, students will be able to:
 
 * define the term "tidy data"
```

Comment:
We should use `-` for this list as we do in earlier chapters . This inconsistency is probably my fault from a long time ago, sorry!

Reply:
no worries! Easy to fix!